final_prep/mod_zero_hero.rb

```diff
@@ -55,17 +116,63 @@
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  'name' => 'Cthulhu',
+  'smell' => 'earthy',
+  'weight' => 10_000,
+  'cities_destroyed' => ['Vancouver', 'London', 'Miami'],
+  'lucky_numbers' => [333, 10, 7],
+  'address' => {
+    'number' => 3799,
+    'street' => 'S Las Vegas Blvd',
+    'state' => 'Nevada',
+    'zip' => 89109
+  }
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age, :arch_nemesis, :power_level, :energy_level
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+end
+
+  def say_name
+    puts @name
+  end
+
+  def maximize_energy
+  puts @energy_level = 1000
```

Comment:
Often, in class methods, you can skip having a `puts` unless you need that feedback. With these methods, we are mostly concerned with making the attribute value change, and we do not necessarily need to see anything given back to us.